addons/hr_homeworking/tests/test_calendar_event.py

```diff
@@ -0,0 +1,216 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.hr_homeworking.tests.common import TestHrHomeworkingCommon
+
+from odoo.tests import tagged
+import datetime
+
+@tagged('post_install', '-at_install', "abc")
+class TestHrHomeworkingHrHomework(TestHrHomeworkingCommon):
+    @classmethod
+    def setUpClass(cls):
+        super(TestHrHomeworkingHrHomework, cls).setUpClass()
+        cls.HrHomework = cls.env['hr.homework']
+
+    def test_action_register_day_work_location(self):
+        one_day_no_weekly_record = self.HrHomework.create({
+            'start_date': "2018-10-16",
+            'stop_date': "2018-10-16",
+            'weekly': False,
+            'work_location_id': self.work_location1.id
+        })
+        one_day_no_weekly_record.with_user(self.user_hruser).action_register_day_work_location()
+        self.assertTrue(one_day_no_weekly_record.meeting_id.homework)
+        self.assertEqual(one_day_no_weekly_record.meeting_id.start_date, datetime.date(2018, 10, 16))
+        self.assertEqual(one_day_no_weekly_record.meeting_id.stop_date, datetime.date(2018, 10, 16))
+        self.assertEqual(one_day_no_weekly_record.meeting_id.name, "office - Bureau 1")
+        self.assertFalse(one_day_no_weekly_record.meeting_id.recurrency)
+        employee_hruser_id = self.env['hr.employee'].search([('user_id', '=',self.user_hruser.id)])
+        self.assertFalse(employee_hruser_id.monday_location_id)
+        self.assertFalse(employee_hruser_id.tuesday_location_id)
+        self.assertFalse(employee_hruser_id.wednesday_location_id)
+        self.assertFalse(employee_hruser_id.thursday_location_id)
+        self.assertFalse(employee_hruser_id.friday_location_id)
+        self.assertFalse(employee_hruser_id.saturday_location_id)
+        self.assertFalse(employee_hruser_id.sunday_location_id)
```

Comment:
you could maybe define a method inside your test class that does all the checks using 
```py
def _check_employee_homeworking(employee, day_values):
    self.assertEqual(employee.monday_location_id, day_values[0])
    self.assertEqual(employee.tuesday_location_id, day_values[1])
    ...
```
then check all your values doing
```py
self._check_employee_homeworking(employee_hruser, [False, False, False, False, False, False, False])
```
And same for your homework record